### ISSUE TITLE
Failed build

### ISSUE THREAD

**Issue by dva** [beltex/dshb#30]
latest commit: ee48c78

> ** BUILD FAILED **
> The following build commands failed:
	CompileSwift normal x86_64
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
make: *** [build] Error 65

**Comment by beltex** [comment]
Hi @dva!

What version of the Swift compiler are you using? **Xcode 7 Beta 2** is expected.

```sh
$ swiftc -v
Apple Swift version 2.0 (swiftlang-700.0.42.3 clang-700.0.53.3)
Target: x86_64-apple-darwin14.3.0
```

Can you share the full log by any chance?

**Comment by dva** [comment]
Hey @beltex.

Swift compiler:
```
Apple Swift version 1.2 (swiftlang-602.0.53.1 clang-602.0.53)
Target: x86_64-apple-darwin14.4.0
```

Full build log: https://gist.github.com/dva/1a4cb60e4ba8e3607dea

P.S.: I haven't seen you made switch to **Xcode 7 Beta 2**.

**Comment by beltex** [comment]
ah, yeah thats it then. I merged the Swift 2.0 branch into master this morning, apologies for not opening a pull request before doing so to make it clearer. As for why, pushing forward with a beta version of Xcode/Swift, since it won't go GM till around September/October (but stable still) & **dshb** doesn't have a large user base at this point, I thought might as well.

You can download **Xcode 7 Beta 3** [here](https://developer.apple.com/xcode/downloads/) (beta 2 is no longer available - beta 3 works fine though).

**Comment by dva** [comment]
@beltex ok, thanks! Clear for now.
